force-app/main/default/lwc/paycheckCalculator/paycheckCalculator.js

```diff
@@ -0,0 +1,119 @@
+import { LightningElement, track } from 'lwc';
+
+export default class PaycheckCalculator extends LightningElement {
+
+    annualSalary = 0;
+    monthlyGrossSalary = 0;
+    biWeeklyGrossSalary = 0;
+    monthlyNetSalary = 0;
+    biWeeklyNetSalary = 0;
+
+    @track federalIncomeTax = 0;
+    @track socialSecurityTax = 0;
+    @track medicareWithholding = 0;
+    @track takeHomePay = 0;
+
+    federalTaxBrackets = [
+        { limit: 11000, rate: 0.1 },
```

Comment:
Consider leveraging custom metadata to store tax bracket info so that you can easily update on a yearly basis without needing a code change